app/helpers/tv_series_helper.rb

```diff
@@ -8,4 +7,13 @@ def image_for_tv_poster(series)
       render "shared/missing_poster", title: title
     end
   end
+
+  def formatted_season_number(season_number)
```

Comment:
I moved this display logic out of the `tmdb_handler#tmdb_handler_tv_season` method as part of dissolving instance variables.